### DISCUSSION TITLE
Help in entering the equation

### DISCUSSION THREAD

**Discussion by ehsanps4** [WannesMalfait/Blender-Add-ons#29] [General]
Hello. Thank you very much for the add-on. With the help of SVG to equations software, the equation of a circle is available for example. If possible, guide the steps to enter equations like this in the Material Blender section. Thankful.

SVG to equations:
https://github.com/maltaisn/svg-equations

Circle equation:
[01.txt](https://github.com/WannesMalfait/Blender-Add-ons/files/14214353/01.txt)

![01](https://github.com/WannesMalfait/Blender-Add-ons/assets/85092258/9255337e-e29b-4206-9cd2-3a65f0a76463)

**Comment by WannesMalfait** [comment]
You can create a circle like that using the following formula in shader nodes:
```
tex_coords().object.length() > 0.8
```
![image](https://github.com/WannesMalfait/Blender-Add-ons/assets/46323945/0bad0bad-0c4b-4ddf-bc22-6ef0c9157336)

**Reply by ehsanps4** [reply]
Thankful. I mean using SVG to equations software to import different shapes. Help to enter the output formulas of the SVG to equations software. Please guide the steps to enter equations like this. Thank you.

Example equation:
[01.txt](https://github.com/WannesMalfait/Blender-Add-ons/files/14229579/01.txt)

**Comment by WannesMalfait** [comment]
Those types of equations are not really suited for shaders, sadly. There is no good way to use those formulas with my add-on

**Reply by ehsanps4** [reply]
Thank you.
